docs/configuration.md

```diff
@@ -0,0 +1,541 @@
+Command line: `N/A`  
+Config file: `"disable-bail": true`
+
+Stryker aborts a unit testrun for a mutant as soon as one test fails because this is enough to confirm the mutant is killed. This can reduce the total runtime but also means you miss information about individual unit tests (eg if a unit test does not kill any mutants and is therefore useless). You can disable this behavior and run all unit tests for a mutant to completion. This can be especially useful when you want to find useless unit tests.
+
+### `disable-mix-mutants` <`bool`>
+
+Default: `false`  
+Command line: `N/A`  
+Config file: `"disable-mix-mutants": true`
+
+Stryker combines multiple mutants in the same testrun when the mutants are not covered by the same unit tests. This reduces the total runtime. You can disable this behavior and run every mutation in an isolated testrun. This can be useful when mixed mutants have unintended side effects.
+
+### `since` <`flag`>
+
+Default: `false`  
+Command line: `--since:feat-2`  
+Config file: `"since": { }`
+
+Use git information to test only code changes since the given target. Stryker will only report on mutants within the changed code. All other mutants will not have a result.
+
+If you wish to test only changed sources and tests but would like to have a complete mutation report see [with-baseline](#with-baseline-<flag>).
+
+Set the diffing target on the command line by passing a comittish with the since flag in the format `--since:<comittish>`.
+Set the diffing target in the config file by setting the [since.target](#since.target-<comittish>) option.
+
+*\* For changes on test project files all mutants covered by tests in that file will be seen as changed.*
+
+### `since.enabled` <`bool`>
+
+Default: `null`  
+Command line: `N/A`  
+Config file: `"since": { "enabled": false }`
+
+Enable or disable [since](#since-<flag>). If the enabled property is not set but the `since` object exists in the config file it is assumed to be enabled. Use this option to (temporarily) disable `since` without having to delete the other `since` configuration.
+
+### `since.target` <`comittish`>
+
+Default: `master`  
+Command line: `N/A`  
+Config file: `"since": { "target": 'feat-2' }`
+
+Set the diffing target for the [since](#since-<flag>) feature.
+
+### `since.ignore-changes-in` <`string[]`>
+
+Default: `null`  
+Command line: `N/A`  
+Config file: `"since": { "ignore-changes-in: ['**/*Translations.json'] }`
+
+Allows to specify an array of files which should be ignored if present in the diff.
+This feature is only recommended when you are sure these files will not affect results, or when you are prepared to sacrifice accuracy for performance.
+            
+Use [globbing syntax](https://en.wikipedia.org/wiki/Glob_(programming)) for wildcards. Example: ['**/*Assets.json','**/favicon.ico']
+
+# Experimental
+
+**The features in this section are experimental. Results can contain false postives and false negatives.**
+
+## Baseline
```

Comment:
I think we should add example config files fully configured for different scenarios. We could add them in a seperate "page"/ file. Or we could add small examples at the start of each heading in this file

Reply:
For the Baseline feature specifically or for all options?

Reply:
For all options. Just like strykerJs have. I think I have a preference for a seperate page with config examples. And then link to the corresponding example for each category in this file.

Reply:
I personally don't see the point, we don't have that many different scenarios. StrykerJS has completely different setups depending on used frameworks. For us the biggest relevance is whether you use dotnet framework or dotnet core and the only difference is the solution file is required for dotnet framework and optional for dotnet core.

In any case, I would like to see this as a new issue. I don't think it's neccesary for this PR.